.circleci/config.yml

```diff
@@ -19,9 +19,18 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Run python tests
+          name: Install requirements
           command: |
             pip install -r requirements.txt
+
+      - run:
+          name: Run mypy
+          command: |
+            mypy phabricatoremails/cli.py
```

Comment:
Is this only going to check a single module? (i.e. `cli`?)

Reply:
`mypy` works recursively - I've pointed it at `cli`, which is our entry point, then `mypy` will follow the imports and continue validating types until it's out of new files out to check.

Reply:
nit: I think this is utilizing the default import discovery behaviour (`--follow-imports normal`), and not recursion. `mypy phabricatoremails` would run recursively, as well as follow imports, which would be a superset of what import discovery would do. Both of these are equivalent at the moment, though conceivably a change to an import, or adding a new module may cause some modules to not be followed.

Reply:
Yeah, you're right that this discovers files via imports, rather than by the contents of a directory.
When we add new entry points, I think that we should update this accordingly